core/node/consensus/src/storage/store.rs

```diff
@@ -203,93 +213,130 @@ impl StoreRunner {
                 const POLL_INTERVAL: time::Duration = time::Duration::seconds(1);
                 let mut next_batch_number = { batches_persisted.borrow().next() };
                 loop {
-                    let mut conn = pool.connection(ctx).await?;
-                    if let Some(last_batch_number) = conn
-                        .get_last_batch_number(ctx)
-                        .await
-                        .wrap("last_batch_number()")?
-                    {
-                        if last_batch_number >= next_batch_number {
-                            let range = conn.batches_range(ctx).await.wrap("batches_range()")?;
-                            next_batch_number = last_batch_number.next();
-                            batches_persisted.send_replace(range);
+                    let span = tracing::info_span!("gossip_sync_batches_loop", %next_batch_number);
+                    async {
+                        let mut conn = pool.connection(ctx).await?;
+                        if let Some(last_batch_number) = conn
+                            .get_last_batch_number(ctx)
+                            .await
+                            .wrap("last_batch_number()")?
+                        {
+                            if last_batch_number >= next_batch_number {
+                                let range =
+                                    conn.batches_range(ctx).await.wrap("batches_range()")?;
+                                next_batch_number = last_batch_number.next();
+                                tracing::info_span!("batches_persisted_send").in_scope(|| {
+                                    batches_persisted.send_replace(range);
+                                });
+                            }
                         }
+                        ctx.sleep(POLL_INTERVAL)
+                            .instrument(tracing::info_span!("sleep"))
```

Comment:
Again, not strictly followed right now, but I've already noticed that similar spans make the analysis in fact harder. Probably makes sense to make it `debug_span`

Reply:
Looking at this one more time I think this particular sleep shouldn't be covered at all. Other sleep are rare (only if a node is behind) so I think it's fine to keep it as `INFO` for now.